packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/en.ftl

```diff
@@ -1,8 +1,6 @@
 # Variables:
 #  $clientName (String) - A client the user hasn't signed into before (e.g. Firefox, Sync)
-verifyLogin-title = New sign-in to { $clientName }
-verifyLogin-description = For added security, please confirm this sign-in from the following device:
-# Variables:
-#  $clientName (String) - A client the user hasn't signed into before (e.g. Firefox, Sync)
-verifyLogin-subject = Confirm new sign-in to { $clientName }
-verifyLogin-action = Confirm sign-in
+verifyLogin-title-2 = Did you sign in to { $clientName }?
+verifyLogin-description-2 = Help us keep your account safe by confirming you signed in on:
+verifyLogin-subject-2 = Confirm sign in
+verifyLogin-action-2 = Confirm sign in
```

Comment:
@chenba beat me to it, but this line should also be "sign-in".

Reply:
Updated